code/datums/components/suit_autodoc.dm

```diff
@@ -169,7 +169,8 @@
 	SIGNAL_HANDLER
 	if(!iscarbon(user))
 		return
-	UnregisterSignal(wearer, COMSIG_KB_SUITANALYZER)
+	if(wearer)
+		UnregisterSignal(wearer, COMSIG_KB_SUITANALYZER)
```

Comment:
why?

Reply:
I mentioned it in the pr description. It was a runtime that was happening with valk and is current. I know it wasn't atomized but it was a single line fix and it was bugging me when testing.